cms/djangoapps/contentstore/tests/test_bulkupdate.py

```diff
@@ -0,0 +1,124 @@
+"""
+Unit tests for bulk update problem settings utility
+"""
+
+import ddt
+import mock
+
+from xmodule.modulestore.tests.factories import CourseFactory
+
+from cms.djangoapps.contentstore.tests.utils import CourseTestCase
+from contentstore.utils import reverse_course_url
+
+
+SHOW_ANSWER_OPTIONS = [
```

Comment:
Can you import HOW_ANSWER_OPTIONS from  cms/djangoapps/contentstore/views/utilities/bulkupdate.py?

Reply:
done